packages/react-native/src/primitives/Label/Label.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Text } from 'react-native';
+import { LabelProps } from './types';
+import { styles } from './styles';
+
+export default function Label({
+  children,
+  style,
+  ...rest
+}: LabelProps): JSX.Element {
+  return (
+    <Text {...rest} style={[styles.label, style]}>
```

Comment:
Should we add a value for `accessibilityRole`?

Reply:
Perhaps [text](https://reactnative.dev/docs/accessibility#accessibilityrole:~:text=text%20Used%20when%20the%20element%20should%20be%20treated%20as%20static%20text%20that%20cannot%20change.)?